concrete/src/Http/Client/Client.php

```diff
@@ -46,7 +48,21 @@ public function send(ZendRequest $request = null)
         $response = parent::send($request);
         $logger = $this->getLogger();
         if ($logger !== null) {
-            $logger->logResponse($response->getStatusCode(), $response->getHeaders()->toArray(), $response->getBody());
+            $statusCode = $response->getStatusCode();
+            $body = $response->getBody();
+            if (mb_strlen($body) <= 200) {
+                $shortBody = $body;
+            } else {
+                $shortBody = mb_substr($body, 0, 197) . '...';
+            }
+            $logger->debug(
+                "The response code was $statusCode and the body was $shortBody",
```

Comment:
PSR-3 provides automatic interpolation based on the context. This should be `"The response code was {statusCode} and the body was {shortBody}"`